example_tests/value_generation/body_schema.jkt

```diff
@@ -0,0 +1,13 @@
+name: Body Schema
+request:
+  method: Post
+  url: https://api.postcodes.io/postcodes
+  bodySchema:
+    type: object
+    schema: 
+        postcodes: 
+            type: list
```

Comment:
is it feasible to collapse these? Basically I feel like list[type] or list<type> is cleaner than list with another nesting layer of schema for the other type. I guess the nested schema provides more flexibility if you need to give ranges and stuff for variable generation? Maybe it's fine as it is.

Reply:
I think you need it to allow you to specify some things clearly for checking and validation. However, it could be added later at the cost of some expressiveness.